lib/matplotlib/__init__.py

```diff
@@ -207,14 +207,35 @@ def _check_versions():
     sys.argv = ['modpython']
 
 
-_verbose_msg = """\
-matplotlib.verbose is deprecated;
-Command line argument --verbose-LEVEL is deprecated.
-This functionality is now provided by the standard
-python logging library.  To get more (or less) logging output:
-    import logging
-    logger = logging.getLogger('matplotlib')
-    logger.set_level(logging.INFO)"""
+def verbose(level='debug'):
```

Comment:
This seems largely redundant with `_set_logger_verbose_level`. Wouldn't it be sufficient to have one public function?

Reply:
Maybe - I wasnt' aware that the command line handling was gone in master when I did this.  If we really think the commad line handling can go away, then I agree.  But I'm not sure we should fully drop the commandline option.  

If we merge, do we want to keep the `filestr` argument to allow re-direct of the log?  

The point here was to give naive users an easy way to spit out verbose info so I'm not convnced a file redirect is a good idea.

Reply:
Didn’t erase the function, but don’t use it anymore...  I didn’t erase just because I’m not sure I could recreate the code from scratch...